### DISCUSSION TITLE
Keyboard shortcuts for formatting

### DISCUSSION THREAD

**Discussion by adityamiskin** [mdx-js/mdx#2479] [Q&A]
How do I setup mdx extension for keyboard shortcuts for bold, italics and other stuff?? I dont see anything in the documentation.

**Comment by ChristianMurphy** [comment]
Welcome @adityamiskin! 👋 
The default keybindings match what you'd find in other word processors.
```json
"keybindings": [
  {
    "command": "mdx.toggleEmphasis",
    "key": "ctrl+i",
    "mac": "cmd+i",
    "when": "editorTextFocus && !editorReadonly && editorLangId == mdx"
  },
  {
    "command": "mdx.toggleDelete",
    "key": "alt+shift+5",
    "mac": "cmd+shift+x",
    "when": "editorTextFocus && !editorReadonly && editorLangId == mdx"
  },
  {
    "command": "mdx.toggleStrong",
    "key": "ctrl+b",
    "mac": "cmd+b",
    "when": "editorTextFocus && !editorReadonly && editorLangId == mdx"
  }
],
```

source: https://github.com/mdx-js/mdx-analyzer/blob/7b7c0ae49b3e8d6a413cc694b1a590d712c97fde/packages/vscode-mdx/package.json#L157-L176

You can also adjust them if you prefer different bindings https://code.visualstudio.com/docs/getstarted/keybindings
